WordPress/src/main/res/layout/plugin_browser_activity.xml

```diff
@@ -74,6 +74,61 @@
                 </RelativeLayout>
             </android.support.v7.widget.CardView>
 
+            <android.support.v7.widget.CardView
+                android:id="@+id/featured_plugins_cardview"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/margin_extra_large"
+                android:visibility="gone"
+                card_view:cardBackgroundColor="@color/white"
+                card_view:cardCornerRadius="@dimen/cardview_default_radius"
+                tools:visibility="visible">
+
+                <RelativeLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content">
+
+                    <TextView
+                        android:id="@+id/text_featured"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_marginLeft="@dimen/margin_extra_large"
+                        android:layout_marginStart="@dimen/margin_extra_large"
+                        android:layout_marginTop="@dimen/margin_extra_large"
+                        android:text="@string/plugin_caption_featured"
```

Comment:
Although fairly unlikely, it's possible that `text_featured` and `text_all_featured` can overlap.  To avoid that, we can add the following to the `text_featured` view.

``` xml
android:ellipsize="end"
android:layout_alignParentLeft="true"
android:layout_alignParentStart="true"
android:layout_toLeftOf="@+id/text_all_featured"
android:layout_toStartOf="@+id/text_all_featured"
```

The same attributes can be added to the other three view collections in this layout as well (i.e. `text_installed`, `text_popular`, and `text_new`).

Reply:
@theck13 I've added your suggestion in f78f5170e66daaacd0d090a92353c9ea426d2109. I've also introduced styles for both the title and the button in f78f5170e66daaacd0d090a92353c9ea426d2109 & 69424bcf656e15979ca3443fb0368ac58391535b to clean up the xml a bit. It should be ready for another pass!